cpp/collections_module/algorithm/algorithm.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <mgp.hpp>
+
+namespace Collections{
+
+constexpr const char *kResultSumLongs = "sum";
```

Comment:
`const char * `is a global that may change. use constexpr std::string_view for better code stlye

Reply:
In this exact place kResultSumLongs needs to be char* because field_name in record.Insert() needs to be char*, changed in other places.

Reply:
You can use on that `std::string(kResultSumLongs).c_str()`